test/perl/TestServer.pl

```diff
@@ -99,53 +99,59 @@ sub usage {
     exit 1;
 }
 
-my $handler = new ThriftTestHandler();
-my $handler2 = new SecondServiceHandler();
-my $processor = new ThriftTest::ThriftTestProcessor($handler);
-my $processor2 = new ThriftTest::SecondServiceProcessor($handler2);
+my $handler    = ThriftTestHandler->new();
+my $handler2   = SecondServiceHandler()->new;
```

Comment:
This line is incorrect and causing the cross test feature failure.

Reply:
I have just corrected this, and force pushed the updated commit